backend/subroker/views.py

```diff
@@ -1,3 +1,392 @@
+    if request.method == 'GET':
+        if request.user.is_authenticated:
+            logout(request)
+            return HttpResponse(status=204)
+        else:
+            return HttpResponse(status=401)
+    else:
+        return HttpResponseNotAllowed(['GET'])
+
+#Group
+def group_list(request):    
+    if request.method == 'GET':
+        if request.user.is_authenticated:
+            group_all_list = [group for group in Group.objects.all().values()]
+            if not group_all_list:
+                return HttpResponse(status=404)
+            return JsonResponse(group_all_list, safe=False, status=200)
+        else:
+            return HttpResponse(status=401)
+        
+    elif request.method == 'POST':
+        if request.user.is_authenticated:
+            try:
+                req_data = json.loads(request.body.decode())
+                group_name = req_data['name']
+                group_description = req_data['description']
+                group_is_public = bool(req_data['is_public'])
+                group_password = int(req_data['password'])
+                group_ott = req_data['ott']
+                group_payday = int(req_data['payday'])
+                group_account_bank = req_data['account_bank']
+                group_account_number = req_data['account_number']
+                group_account_name = req_data['account_name']
+                group_leader = request.user
+            except (JSONDecodeError) as e:
+                return HttpResponseBadRequest()
+            try:
+                group_membership = Ott.objects.get(id=req_data['membership'])
+            except (Ott.DoesNotExist) as e:
+                return HttpResponse(status=404)
+            group = Group(
+                name=group_name, 
+                description=group_description, 
+                is_public=group_is_public, 
+                password=group_password, 
+                ott=group_ott,
+                membership=group_membership, 
+                payday=group_payday, 
+                account_bank = group_account_bank, 
+                account_number=group_account_number, 
+                account_name=group_account_name, 
+                leader=group_leader
+                )
+            group.save()
+            response_dict = {'id': group.id, 'name': group.name, 'leader': group.leader.id} 
+            return JsonResponse(response_dict, status=201)
+        else:
+            return HttpResponse(status=401)
+    else:
+        return HttpResponseNotAllowed(['GET', 'POST'])
+
+def group_info(request, group_id):
+    if request.method == 'GET':
+        if request.user.is_authenticated:
+            try:
+                group = Group.objects.get(id=group_id)
+            except (Group.DoesNotExist) as e:
+                return HttpResponse(status=404)
+            member_arr = []
+            members = group.members.all()
+            for member in members:
+                member_arr.append(member.id)
```

Comment:
@muhwagwa I think this is better way to make an array
```suggestion
            members = [member for member in group.members.all()]
```